src/components/MultiHopTrade/components/LimitOrder/hooks/useLimitOrderRecipientAddress.tsx

```diff
@@ -0,0 +1,108 @@
+import type { Asset } from '@shapeshiftoss/types'
+import { useCallback, useMemo, useState } from 'react'
+import { useManualReceiveAddressIsRequired } from 'components/MultiHopTrade/hooks/useManualReceiveAddressIsRequired'
+import { useReceiveAddress } from 'components/MultiHopTrade/hooks/useReceiveAddress'
+
+import { SharedRecipientAddress } from '../../SharedTradeInput/SharedRecipientAddress'
+
+type UseLimitOrderRecipientAddressProps = {
+  buyAsset: Asset
+  buyAccountId: string | undefined
+  sellAccountId: string | undefined
+}
+
+export const useLimitOrderRecipientAddress = ({
+  buyAsset,
+  buyAccountId,
+  sellAccountId,
+}: UseLimitOrderRecipientAddressProps) => {
+  const [manualReceiveAddress, setManualReceiveAddress] = useState<string | undefined>(undefined)
+  const [manualReceiveAddressIsValid, setManualReceiveAddressIsValid] = useState<
```

Comment:
nitpick: and same preference for `is` prefix here and in the editing/validating below

Reply:
ca48fd9fe3fd49f555fe0341091aa2e7844981a8

Reply:
ca48fd9fe3fd49f555fe0341091aa2e7844981a8